src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Signing/TrustedSignersCommand.cs

```diff
@@ -23,107 +22,305 @@ internal static class TrustedSignersCommand
+
+                    CommandOption allowUntrustedRootOption = authorCommand.Option(
+                                                "--allow-untrusted-root",
+                                                Strings.TrustCommandAllowUntrustedRoot,
+                                                CommandOptionType.NoValue);
+
+                    CommandOption configFile = authorCommand.Option(
+                        "--configfile",
+                        Strings.Option_ConfigFile,
+                        CommandOptionType.SingleValue);
+
+                    authorCommand.HelpOption(XPlatUtility.HelpOption);
 
-                CommandOption configFile = trustedSignersCmd.Option(
+                    CommandOption verbosity = authorCommand.VerbosityOption();
+
+                    CommandArgument name = authorCommand.Argument("<NAME>",
+                                               Strings.TrustedSignerNameToAdd);
+                    CommandArgument package = authorCommand.Argument("<PACKAGE>",
+                                               Strings.TrustLocalSignedNupkgPath);
+
+                    authorCommand.OnExecute(async () =>
+                    {
+                        return await ExecuteCommand(TrustCommand.Author, algorithm: null, allowUntrustedRootOption.HasValue(), owners: null, verbosity, configFile, getLogger, setLogLevel, name: name.Value, sourceUrl: null, packagePath: package.Value);
+                    });
+                });
+
+                trustedSignersCmd.Command("repository", (repositoryCommand) =>
+                {
+                    repositoryCommand.Description = Strings.TrustRepositoryCommandDescription;
+
+                    CommandOption allowUntrustedRootOption = repositoryCommand.Option(
+                                                "--allow-untrusted-root",
+                                                Strings.TrustCommandAllowUntrustedRoot,
+                                                CommandOptionType.NoValue);
+
+                    CommandOption configFile = repositoryCommand.Option(
+                        "--configfile",
+                        Strings.Option_ConfigFile,
+                        CommandOptionType.SingleValue);
+
+                    repositoryCommand.HelpOption(XPlatUtility.HelpOption);
+
+                    CommandOption owners = repositoryCommand.Option(
+                        "--owners",
+                        Strings.TrustCommandOwners,
+                        CommandOptionType.MultipleValue);
+
+                    CommandOption verbosity = repositoryCommand.VerbosityOption();
+
+                    CommandArgument name = repositoryCommand.Argument("<NAME>",
+                                               Strings.TrustedSignerNameToAdd);
+                    CommandArgument package = repositoryCommand.Argument("<PACKAGE>",
+                                               Strings.TrustLocalSignedNupkgPath);
```

Comment:
As per [spec](https://github.com/NuGet/Home/blob/b30a8ff077272045224829c6772327b724ebed35/proposed/2021/DotnetNugetTrustedSigners.md#note), `For trust author and trust repository, <package> may be a glob/wildcard. This must be consistent across platforms, so the Windows version must expand these wildcards when it receives them as arguments.`. There was a similar requirement for `dotnet nuget verify` command, hence we passed `multipleValues: true`.

https://github.com/NuGet/NuGet.Client/blob/966b1b21eec28834365a8604744dab682c8326ea/src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Signing/VerifyCommand.cs#L24-L27

Reply:
See https://github.com/NuGet/NuGet.Client/pull/4211#discussion_r720614519